### PR TITLE
feat(milo-modifier): add sb-text--milo modifier

### PR DESCRIPTION
**Pull request by MLKiiwy**
Reviewers: @MLKiiwy @kagnamazian @realdreamer 

Add a new modifier to force using milo.

### CODE REVIEW

File: src/trumps/_text.scss [hunk 1]
```diff
@@ -20,6 +20,9 @@
 // .sb-text--uppercase - Used for uppercase text
 // .sb-text--lowercase - Used for lowercase text
 // .sb-text--number    - Used to reset numbers to their original alignment
+// .sb-text--number    - Used to reset numbers to their original alignment
+// .sb-text--smallcaps - Used to display text in small caps
+// .sb-text--milo      - Force Milo font
```
Comment (kagnamazian) [inline src/trumps/_text.scss]: is it something than Gal ask ?

Comment (MLKiiwy) [reply src/trumps/_text.scss]: it's ask by @Maruceau

Comment (MLKiiwy) [reply src/trumps/_text.scss]: Ok @kagnamazian it's good for now, we can add this modifier. I think later we are going to do a refactoring with less modifier when @Maruceau will be ready

File: src/trumps/_text.scss [hunk 2]
```diff
@@ -20,6 +20,9 @@
 // .sb-text--uppercase - Used for uppercase text
 // .sb-text--lowercase - Used for lowercase text
 // .sb-text--number    - Used to reset numbers to their original alignment
+// .sb-text--number    - Used to reset numbers to their original alignment
```
Comment (dine) [inline src/trumps/_text.scss]: `.sb-text--number` is duplicated :)

**Pull request merged**